### PR TITLE
fix(docs): improved readability in READ.md

### PR DESCRIPTION
**Pull request by rutvikk05**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: web/README.md [hunk 1]
```diff
@@ -24,11 +26,11 @@ http://localhost:3000/responsive-web-design
 
 is the curriculum data that is currently being used.
 
-## Things of Note
+## Things to Note :
```
Comment (jeremylt) [inline web/README.md:29]:
```suggestion
## Things of Note
```

None of the other headings have this extra :

File: web/README.md [hunk 2]
```diff
@@ -24,11 +26,11 @@ http://localhost:3000/responsive-web-design
 
 is the curriculum data that is currently being used.
 
-## Things of Note
+## Things to Note : 
 
-Incremental static regeneration is working quite nicely. You can modify the curriculum data (in /curriculum-server/data/curriculum.json), refresh reload your browser and the changes will be reflected.
+Incremental static regeneration is working quite nicely. You can modify the curriculum data (in /curriculum-server/data/curriculum.json), refresh/reload your browser and the changes will be reflected.
```
Comment (jeremylt) [inline web/README.md:31]: This is perhaps the only change I see as not just a happy-to-glad in this PR

**Review by raisedadead — APPROVED** [review]
#### :sparkles: :sparkles: **AWESOME!** :sparkles: :sparkles: 

LGTM.

Thanks for this pull request, @rutvikk05, and for contributing to the codebase for the first time.

We will be looking forward to more contributions in the future.

Cheers and happy contributing!

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Pull request merged**